citar.el

```diff
@@ -176,18 +176,19 @@ references as a string."
 
 (defcustom citar-display-transform-functions
   ;; TODO change this name, as it might be confusing?
-  '((("author" "editor") . citar--shorten-names))
+  '((t . ((("author" "editor") . (citar--shorten-names))))
+    (notes . ((("author" "editor") . (citar--shorten-names 3 "&")))))
```

Comment:
The doc string here needs to be changed. In particular we need to specify how the arguments will be used. Note that  for value `(citar--shorten-names 3 "&")`,  `citar--shorten-names` will be called as `(citar--shorten-names STR 3 "&")` and this needs to be documented.